pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -14,6 +14,11 @@
     <ShouldCreateLayout>false</ShouldCreateLayout>
     <PropsFile>$(IntermediateOutputPath)$(MSBuildProjectName).props</PropsFile>
     <PlatformManifestFile>$(IntermediateOutputPath)$(MSBuildProjectName).PlatformManifest.txt</PlatformManifestFile>
+    <!-- During an official build when we can guarantee that all RID-specific dependencies have been built,
+         restore all of those dependencies and gather the prospective content of the RID-specific Core.App
+         packages.  This is needed so that we have a complete platform manifest in the shipping version of
+         the Microsoft.NETCore.App (RID-agnostic/identity package). -->
+    <IncludeAllRuntimePackagesInPlatformManifest Condition="'$(SIGNED_PACKAGES)' == 'true'">true</IncludeAllRuntimePackagesInPlatformManifest>
```

Comment:
We're free to pass in IncludeAllRuntimePackagesInPlatformManifest if that's what you want.  I used SIGNED_PACKAGES because this is already passed in and used by the core-setup build to do a similar thing for other packages.